comp/otelcol/collector-contrib/impl/go.mod

```diff
@@ -1,6 +1,6 @@
 // Code generated by "go.opentelemetry.io/collector/cmd/builder". DO NOT EDIT.
 
-module github.com/DataDog/comp/otelcol/collector-contrib
+module github.com/DataDog/datadog-agent/comp/otelcol/collector-contrib
```

Comment:
I searched for [occurrences](https://github.com/search?q=%22github.com%2FDataDog%2Fcomp%2Fotelcol%2Fcollector-contrib%22&type=code) of the module name on Github:
- almost every match is in this repo or in forks
- there is one match in the [documentation repo](https://github.com/DataDog/documentation/blob/f5528ddbba2a105f66f116eafa6004b1b969f281/content/en/opentelemetry/agent/agent_with_custom_components.md?plain=1#L63), I guess it will have to be updated once the module is released
- one in a private Datadog repo which seems to have been used to test the otel agent a few weeks ago

I guess the command was ran manually, the `go.mod` file was updated (eg. the go version was changed several times) and not reverted since.
I'll ask the opentelemetry team to be sure.

Reply:
The code is generated from `manifest.yaml` using the command `inv collector.generate` . Since we updated the manifest, it should be good.